pkg/okteto/preview.go

```diff
@@ -136,6 +143,42 @@ func ListPreviews(ctx context.Context) ([]PreviewEnv, error) {
 	return body.Previews, nil
 }
 
+func ListPreviewsEndpoints(ctx context.Context, previewName string) ([]string, error) {
+	endpoints := make([]string, 0)
+
+	q := fmt.Sprintf(`query{
+		preview(id: "%s"){
+			deployments{
```

Comment:
You need to query `gitDeploys`, `stacks` and `apps` too, and for each of them, `deployments` and `statefulsets` endpoints:
https://github.com/okteto/app/blob/master/backend/pkg/graphql/types.go#L277